### DISCUSSION TITLE
How to add 'highlights' and timestamps

### DISCUSSION THREAD

**Discussion by itzilly** [Eugeny/tabby#9696] [General]
I'm not too sure how to ask this since I don't know what this is called, but is there a way to get this sort of setup in Tabby?
![image](https://github.com/Eugeny/tabby/assets/90023277/203c714c-6d56-45b7-b37e-c1ba72c085be)
Including the optional git stuff when it applies? I'm not sure what this is called so if anyone could tell me that would be fantastic as well.

I'm also wondering if you can add a time to the right side, so you can see when commands were run / how commands are running so I can keep track of the time? For example, every time the latest line updates (in something like installing/downloading) it shows the time it was updated/changed at so if something hangs I can see the time it most recently 'updated'?

Also no discord server for tabby?

**Comment by moemoechu** [comment]
thats not managed by terminal app but for shell itself, check:
[p10k](https://github.com/romkatv/powerlevel10k)
[starship](https://github.com/starship/starship)
or
[oh-my-zsh](https://ohmyz.sh)
